net/crowdsec/files/crowdsec.initd

```diff
@@ -0,0 +1,50 @@
+#!/bin/sh /etc/rc.common
+# (C) 2021 Gerald Kerma
+
+START=99
+USE_PROCD=1
+NAME=crowdsec
+PROG=/usr/bin/crowdsec
+CONFIG=/etc/crowdsec/config.yaml
+RUNCONFDIR=/srv/crowdsec/data
+
+service_triggers() {
+        procd_add_reload_trigger crowdsec
+}
+
+boot()
+{
+	rc_procd start_service
+}
```

Comment:
`boot()` should be unnecessary with `START=99`.

Reply:
removed
thanks